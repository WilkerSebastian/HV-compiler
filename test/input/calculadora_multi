0-9 195 0-1 181 0-2 180 0-0 480 190 890 090 281 480 190 890 081 281 181 095 381 610 000