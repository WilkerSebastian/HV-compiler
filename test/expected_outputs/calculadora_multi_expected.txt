0
2
8
24
64
